docusaurus/docs/publish-a-plugin/distribute-a-plugin.md

```diff
@@ -1,27 +0,0 @@
----
-id: distribute-a-plugin
-title: Distribute a plugin
-sidebar_position: 1
----
-
-To distribute a Grafana plugin either to the community or privately, the plugin must be signed so the Grafana application can verify its authenticity. You can do this with the `@grafana/sign-plugin` package.
-
-:::info
-
-It's not necessary to sign a plugin during development. The [Docker development environment](../get-started/set-up-development-environment.mdx) that is scaffolded with `@grafana/create-plugin` will load the plugin without a signature.
-
-:::
-
-## Initial steps
-
-Before signing a plugin, read the Grafana [plugin publishing and signing criteria](https://grafana.com/docs/grafana/latest/developers/plugins/publishing-and-signing-criteria/) documentation carefully.
-
-`@grafana/create-plugin` adds the necessary commands and github workflows for signing and distributing a plugin via the Grafana plugins catalog.
```

Comment:
We've lost this but it was generally a bit vague. A follow up would be to better link the release workflow documentation and sign/publish documentation

Reply:
Captured in https://github.com/grafana/plugin-tools/issues/360